### PR TITLE
Create 2021-02-02-Outreachy post

### PR DESCRIPTION
**Pull request by viniciusdc**
## how to add a blog post

1. Put your post in `.rst` format in the `posts` directory. Look at the other posts for the format. 
   Make sure you have a `.. post::` directive at the top and a section title at the top.
2. You can build your post via the commands `make clean`, `ablog build`, and `ablog serve`.
3. Checkout the results in your browser.
4. When you are ready, make a PR!

### CODE REVIEW

**Comment by viniciusdc** [comment]
@ocefpaf give a look if you have some time

File: posts/2020-02-04-Outreachy.rst [hunk 1]
```diff
@@ -0,0 +1,210 @@
+.. post:: 2021-02-02
+   :tags: Outreachy
+   :author: viniciusdc
+   :redirect: 2021/02/02/Outreachy
+
+Conda-forge Outreachy
+=====================
+
+COnda-forge is participating for this round of Outreachy. The goal of
```
Comment (ocefpaf) [inline posts/2020-02-04-Outreachy.rst]:
```suggestion
Conda-forge is participating for this round of Outreachy. The goal of
```

File: posts/2020-02-04-Outreachy.rst [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+Participant Application Process:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, please review the Outreachy Eligibility and Application
+Information page to learn more about eligibility for Outreachy.
+
+Steps for applicants to conda-forge:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+| Confirm your eligibility on the outreachy site
```
Comment (ocefpaf) [inline posts/2020-02-04-Outreachy.rst]: I'm not a rst expert but how would these `|` render in the final page?

**Review by ocefpaf — APPROVED** [review]
LGTM, just some minor questions and edits.

**Pull request merged**